### PR TITLE
feat: grant submit changes

### PR DESCRIPTION
**Pull request by andyesp**

### CODE REVIEW

File: src/components/GrantRequest/GrantRequestFundingSection.tsx [hunk 1]
```diff
@@ -142,139 +146,85 @@ export default function GrantRequestFundingSection({ onValidation, isFormDisable
...
+              value={state.value.funding}
+              onChange={({ currentTarget }) =>
+                editor.set({
+                  funding: currentTarget.value,
+                  projectDuration: updateProjectDuration(currentTarget.value, state.value.projectDuration),
+                })
+              }
+              onBlur={() => editor.set({ funding: state.value.funding })}
+              error={state.error.funding || ''}
+              // message={t(state.error.funding)}
```
Comment (1emu) [inline src/components/GrantRequest/GrantRequestFundingSection.tsx:199]: we can remove this :)

File: src/components/GrantRequest/ProjectDurationInput.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  }, [onChange, optionMinLimit, value])
+
+  return (
+    <div>
+      <Label>{t('page.submit_grant.funding_section.project_duration_title')}</Label>
+      <div>
+        <div className="ProjectDurationInput__InputContainer">
+          <div className="ProjectDurationInput__Input">
+            <button className="ProjectDurationInput__Button" onClick={handleRemoveClick}>
+              <Remove />
```
Comment (1emu) [inline src/components/GrantRequest/ProjectDurationInput.tsx:45]: WDYT about increase/decrease instead of add/remove?

**Review by 1emu — APPROVED** [review]
👏👏👏

File: src/components/GrantRequest/ProjectDurationInput.tsx (hunk 2 above)
Comment (andyesp) [reply src/components/GrantRequest/ProjectDurationInput.tsx:45]: I think those are the common names in the Icons Realm. Other possibility could be using Minus / Plus

Comment (andyesp) [reply src/components/GrantRequest/ProjectDurationInput.tsx:45]: I like better Increase/Decrease for this case. But I don't know if it's better to keep them more generic.

**Pull request merged**